openslides_backend/action/actions/meeting/create_update_delete.py

```diff
@@ -104,6 +116,165 @@
+                    "meeting.can_see_frontpage",
+                    "motion.can_see",
+                    "projector.can_see",
+                    "user.can_see",
+                    "user.can_change_own_password",
+                ],
+            },
+            {
+                "name": "Admin",
+                "meeting_id": instance["id"],
+            },
+            {
+                "name": "Delegates",
+                "meeting_id": instance["id"],
+                "permissions": [
+                    "agenda_item.can_see_internal",
+                    "assignment.can_nominate_other",
+                    "assignment.can_nominate_self",
+                    "list_of_speakers.can_be_speaker",
+                    "mediafile.can_see",
+                    "meeting.can_see_autopilot",
+                    "meeting.can_see_frontpage",
+                    "motion.can_create",
+                    "motion.can_create_amendments",
+                    "motion.can_support",
+                    "projector.can_see",
+                    "user.can_see",
+                    "user.can_change_own_password",
+                ],
+            },
+            {
+                "name": "Staff",
+                "meeting_id": instance["id"],
+                "permissions": [
+                    "agenda_item.can_manage",
+                    "assignment.can_manage",
+                    "assignment.can_nominate_self",
+                    "list_of_speakers.can_be_speaker",
+                    "list_of_speakers.can_manage",
+                    "mediafile.can_manage",
+                    "meeting.can_see_frontpage",
+                    "meeting.can_see_history",
+                    "motion.can_manage",
+                    "projector.can_manage",
+                    "tag.can_manage",
+                    "user.can_manage",
+                    "user.can_change_own_password",
+                ],
+            },
+            {
+                "name": "Committees",
+                "meeting_id": instance["id"],
+                "permissions": [
+                    "agenda_item.can_see_internal",
+                    "assignment.can_see",
+                    "list_of_speakers.can_see",
+                    "mediafile.can_see",
+                    "meeting.can_see_frontpage",
+                    "motion.can_create",
+                    "motion.can_create_amendments",
+                    "motion.can_support",
+                    "projector.can_see",
+                    "user.can_see",
+                ],
+            },
+        ]
+        write_requests, action_results = self.execute_other_action(GroupCreate, payload)
+
+        used_groups_dict = {
+            event["fields"]["name"]: event["fqid"].id  # type: ignore
+            for event in write_requests.events  # type: ignore
+            if event["type"] == EventType.Create
+            and event["fields"]["name"] in ("Admin", "Default")  # type: ignore
+        }
```

Comment:
Just fetch the group ids from the `action_results`, they are always in the same order as the given payload. That way you don't have to search through the write requests.

Reply:
Proposal from Finn: I wrote a [new issue](https://github.com/OpenSlides/openslides-backend/issues/520) to discuss the action_result/write_request theme and we merge this version for now.